pytorch/training/docker/1.11/py3/cu113/Dockerfile.gpu

```diff
@@ -1,6 +1,7 @@
 ARG PYTHON=python3
 ARG PYTHON_VERSION=3.8.12
 ARG PYTHON_SHORT_VERSION=3.8
+ARG MAMBA_VERSION=4.12.0-0
```

Comment:
nit: Please make the `MAMBA_VERSION` ARG definition consistent across both the CPU and the GPU dockerfiles, by either defining it globally, or by defining it specifically in the `common` section.